README.md

```diff
@@ -19,8 +19,7 @@ Code documentation is automatically generated using Doxygen tool, available at [
 ## Build
 
 Build guides for no-OS projects:
-  * [Linux](https://github.com/analogdevicesinc/no-OS/wiki/Building-no-OS-on-Linux)
-  * [Windows](https://github.com/analogdevicesinc/no-OS/wiki/Building-no-OS-on-Windows)
+  * See [Wiki](https://github.com/analogdevicesinc/no-OS/wiki)
```

Comment:
nitpick: I would remove `See` and be more specific that these are Github Wiki pages and not analog wiki pages.

There is another `wiki` link above in the README file that redirects the user to wiki.analog.com

Reply:
Why replace the Linux/Windows links with a top level link ?
I think users want this kind of direct link based on their OS specific build guide.
And the OS specific guide should contain instructions for all platforms. This means that ADUCM and STM32 specific stuff should ultimately end up in those 2 build guides.

Reply:
Because beside the path configuration scripts, the makefile is OS independent now.

Reply:
commit removed